web/translation/translate.en_US.toml

```diff
@@ -75,31 +75,31 @@
 
 [pages.index]
 "title" = "System Status"
-"memory" = "Memory"
-"hard" = "Hard Disk"
+"memory" = "RAM"
+"hard" = "Disk"
 "xrayStatus" = "Xray Status"
 "stopXray" = "Stop"
 "restartXray" = "Restart"
 "xraySwitch" = "Switch Version"
-"xraySwitchClick" = "Choose the version you want to switch to."
-"xraySwitchClickDesk" = "Choose wisely, as older versions may not be compatible with current configurations."
-"operationHours" = "Operation Hours"
-"operationHoursDesc" = "System uptime: time since startup."
+"xraySwitchClick" = "Switch Xray-Core Running Version"
```

Comment:
I don't like this one as well :/
It does not demonstrate what we are going to provide in that Modal!